src/text/raw_text_writer.rs

```diff
@@ -679,10 +679,10 @@ impl<W: Write> IonWriter for RawTextWriter<W> {
 
     fn parent_type(&self) -> Option<IonType> {
```

Comment:
🗺️ The `RawTextWriter` will be removed in a near-future PR, so I'm not worried about changing this `Option<IonType>` to a `ParentType`.